homeassistant/components/lutron.py

```diff
@@ -0,0 +1,79 @@
+"""
+Component for interacting with a Lutron RadioRA 2 system.
+
+Uses pylutron (http://github.com/thecynic/pylutron).
+"""
+
+import logging
+
+from homeassistant.components import group
+from homeassistant.helpers import discovery
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['https://github.com/thecynic/pylutron/archive/v0.1.0.zip#'
+                'pylutron==0.1.0']
+
+DOMAIN = "lutron"
+
+_LOGGER = logging.getLogger(__name__)
+
+LUTRON_CONTROLLER = 'lutron_controller'
+LUTRON_DEVICES = 'lutron_devices'
+LUTRON_GROUPS = 'lutron_groups'
+
+
+def setup(hass, base_config):
+    """Setup the Lutron component."""
+    from pylutron import Lutron
+
+    hass.data[LUTRON_CONTROLLER] = None
+    hass.data[LUTRON_DEVICES] = {'light': []}
+    hass.data[LUTRON_GROUPS] = {}
+
+    config = base_config.get(DOMAIN)
+    hass.data[LUTRON_CONTROLLER] = Lutron(
+        config['lutron_host'],
+        config['lutron_user'],
+        config['lutron_password']
+    )
+    hass.data[LUTRON_CONTROLLER].load_xml_db()
+    hass.data[LUTRON_CONTROLLER].connect()
+    _LOGGER.info("Connected to Main Repeater @ %s", config['lutron_host'])
+
+    # Sort our devices into types
+    for area in hass.data[LUTRON_CONTROLLER].areas:
+        if area.name not in hass.data[LUTRON_GROUPS]:
+            grp = group.Group.create_group(hass, area.name, [])
+            hass.data[LUTRON_GROUPS][area.name] = grp
+        for output in area.outputs:
+            hass.data[LUTRON_DEVICES]['light'].append((area.name, output))
```

Comment:
Sounds good, thanks. I'll have to rework pylutron a bit to support that. Shouldn't be too bad. It'll go on my TODO list.